app/assets/stylesheets/application.scss

```diff
@@ -178,7 +203,12 @@
 }
 
 .box-background {
-  background-color: $pastel-grey;
+  background-color: $navy-blue;
+  border: 2px solid $navy-blue;
+
+  p {
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.